configure

```diff
@@ -454,18 +455,24 @@ LIBZIP="-lrz"; fi
 else
 USE_LIB_ZIP="0"
 LIBZIP="-lrz"; fi
+
+if [ "$HAVE_LIB_LZ4" = "1" ]; then
```

Comment:
Nope. Dont add system library dependencies. Make lz4 smaller and make it small. Drop that configure patch plz

Reply:
Also thats wrong libzip is not lz4